backend/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -166,9 +163,9 @@ def __init__(self, root, url, org):
            self.urls[0] += '/'
        # Make sure root paths are created
        if not os.path.isdir(self.root):
-           fileutils.makedirs(self.root, user=APACHE_USER, group=APACHE_GROUP)
+           fileutils.makedirs(self.root, user=CFG.httpd_user, group=CFG.httpd_group)
        if not os.path.isdir(self.pkgdir):
-           fileutils.makedirs(self.pkgdir, user=APACHE_USER, group=APACHE_GROUP)
+           fileutils.makedirs(self.pkgdir, user=CFG.httpd_user, group=CFG.httpd_group)
```

Comment:
Apparently I forgot to remove the `CFG` import here. Please move these in an `cfg_component()` block.